udata_gouvfr/theme/templates/banners/banner-2020-06-datactivist-survey.html

```diff
@@ -0,0 +1,30 @@
+<nav class="navbar navbar-static-top navbar-subnav">
+
+    <div class="container banner">
+
+        <p class="text-center text-white">
+          Les portails 
+          <a 
+            class="force-underline"
+            href="https://www.data.gouv.fr/fr/" 
+            target="_blank" 
+          >
+            data.gouv.fr
+          </a>
+          ont besoin de vous !
+          Prenez quelques minutes pour répondre 
+          à notre enquête usagers.
+
+          <a 
+            role="button" 
+            class="btn btn-light no-underline"
+            href="https://docs.google.com/forms/d/1u3q_AkPScHjJiGds6OXfjqLYrM-7tNek98Vb26Q_1BU/prefill"
+            target="_blank"
```

Comment:
add `rel=”noopener noreferrer”` for security reasons, or even remove `target="_blank"` for UX reasons.